contracts/tasks/e2e.ts

```diff
@@ -0,0 +1,162 @@
+import {task} from "hardhat/config";
+import {HardhatRuntimeEnvironment} from "hardhat/types";
+
+const TIMEOUT_SECONDS: number = 300
+const green = "\x1b[32m"
+
+// npx hardhat e2e --contract-address 0xa85233C63b9Ee964Add6F2cffe00Fd84eb32338f --oracle-address 0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e --network localhost
```

Comment:
this is how you would run it, or can have a basic TS script that runs it with .env values etc, however we wish